.flake8

```diff
@@ -0,0 +1,34 @@
+[flake8]
+dictionaries=en_US,python,technical
+docstring-convention = google
+extend-ignore = E203, D107, D200, D203, D205,
+    D212, E231, D400, D401, D410, D411, D412,
+    D413, D414, D415, D416, N806
+    # Ignore E203: Whitespace before ':'
+    # Ignore D107: Missing docstring in __init__
+    # Ignore D200: One-line docstring should fit on one line with quotes
+    # Ignore D203: 1 blank line required before class docstring
+    # Ignore D205: blank line required between summary line and description
+    # Ignore D212: Multi-line docstring summary should start at the first line
+    # Ignore E231: Missing whitespace after ':'
+    # Ignore D400: First line should end with a period
+    # Ignore D401: First line should be in imperative mood
+    # Ignore D410: Missing blank line after section
+    # Ignore D411: Missing blank line before section
+    # Ignore D412: No blank lines allowed between a section header and its content
+    # Ignore D413: Missing blank line after last section
+    # Ignore D414: Section has no content
+    # Ignore D415: First line should end with a period, question mark, or exclamation point
+    # Ignore D416: Section name should end with a colon
+    # Ignore N806: Variable names with all caps (ALL_CAPS)
+max-line-length = 99
+per-file-ignore =
+    tests/evm_transition_tool/test_evaluate.py:E501
+
+extend-exclude =
+    setup.py
+    src/evm_transition_tool/tests/
+    src/ethereum_test_tools/tests/
+    src/ethereum_test_forks/tests/
+
+# vim: set ft=dosini:
```

Comment:
Do we need these modelines?

Reply:
Well spotted, thanks. That snuck over from `setup.cfg`, lightclient added them in the initial setup (99bfc0b30f2fd235ffa349606605ce9a9683c6fb). I don't have anything against these (limited to a few places, at least) if they help people's editing experience, but as it was intended for `.cfg`, I think it's save to remove them here. Done in bc50330b0.